### PR TITLE
Add a marker for unspecified country and use a bar chart instead

### PR DESCRIPTION
**Pull request by mattandrews**
image

### CODE REVIEW

File: controllers/tools/views/applications.njk [hunk 1]
```diff
@@ -161,7 +161,8 @@
                                                 label: '{{ country.title }}',
                                                 data: {{ country.data | dump(2) | safe }},
                                                 borderColor: '{{ country.colour }}',
-                                                fill: '{{ country.colour }}'
+                                                fill: '{{ country.colour }}',
```
Comment (davidrapson) [inline controllers/tools/views/applications.njk]: Do we still need `fill`?

**Comment by davidrapson** [comment]
Side question: Should we make the submitted applications chart a bar as well?

**Comment by mattandrews** [comment]
It uses the same code so it gets these changes too!

File: controllers/tools/views/applications.njk (hunk 1 above)
Comment (mattandrews) [reply controllers/tools/views/applications.njk]: Possibly not but I left it in in case we change back and this way it'll still work with the line format.

**Pull request merged**